contracts/DxMgnPool.sol

```diff
@@ -41,6 +57,19 @@ contract DxMgnPool {
         require(depositToken.transferFrom(msg.sender, address(this), amount), "Failed to transfer deposit");
     }
 
+    function participateInAuction() public {
+        require(!poolingEnded(), "Pooling period is over.");
```

Comment:
Could we turn this into a state machine?
I propose the following states:

	enum States{
		InitizialationsPeriod, // just funding no trading
		TradingPeriod,
		UnlockMGNPeriod /// trading will be halted
	}

Then we can use these modifiers:

modifier isState(Stages _s) { 
		require (state >= _s); 
		_; 
	}

I like the pattern a lot. Not sure what you think about it

Reply:
What's the difference between InitizialationsPeriod and TradingPeriod? I think we will start trading right away as soon as the contract is deployed and people can deposit, while it is trading.

Happy to do a state machine as soon as there are three states.

Reply:
I thought of an initialization period before the 18th. We would only start to trade on the 18th, but would allow deposits even before that.
But okay, let's skip it. We will anyways just deploy last minute :dagger: